requirements.txt

```diff
@@ -0,0 +1 @@
+flake8==3.5.0
```

Comment:
We should not have a `requirements.txt` file in this project. We use `setup.py` instead.